script/spawn-test-deps.sh

```diff
@@ -2,14 +2,15 @@
 #
 # Bitcoind
 #
+
 set -e
 
-source ./script/export-test-envs.sh
+. ./script/export-test-envs.sh
 
 echo "starting bitcoind..."
 bitcoind -datadir=$BTCD_DIR > "$BTCD_DIR/stdout.log" 2>&1 &
 alias bitcoin-cli="bitcoin-cli -rpcwait -datadir=$BTCD_DIR -rpcport=18443"
-bitcoin-cli generatetoaddress 1 "$(bitcoin-cli getnewaddress)" && true
+#bitcoin-cli generatetoaddress 1 "$(bitcoin-cli getnewaddress)" && true
```

Comment:
but how we can generate new blocks then?

Reply:
Problem is not with `generatetoaddress` - so generally speaking we can mine. Problem with `getnewaddress` because bitcoind is not creating default wallet now. Lnd will generate wallets and addresses, and we will use bitcoind for mining. So I think everything should work.